sample.coffee

```diff
@@ -19,14 +19,17 @@ myData = FileCollection({
 
 if Meteor.isClient
 
+   # This assigns a file drop zone to the "file table"
+   # once DOM is ready so jQuery can see it
+   Template.collTest.onRendered ->
```

Comment:
Interesting question. Since the initialization code for resumableJS used to be in the `Meteor.startup` block, which is supposed to mean [the DOM is ready](http://docs.meteor.com/#/full/meteor_startup), I didn't think that nesting it back in there would help.  Empirically, my placement works in Chrome and IE and I've used `onRendered` in the past when packages required initialization.

Nevertheless, I did some research and in fact, this [SO post](https://forums.meteor.com/t/solved-when-should-we-use-meteor-startup/2710) discusses that even 'DOM ready' doesn't mean that templates are ready.  I guess the the definitive answer is in the [onRendered source code](https://github.com/meteor/meteor/blob/master/packages/blaze/template.js#L81): 

> Register a function to be called when an instance of this template is inserted into the DOM.

So I think Meteor is ensuring that the function only runs when the template exists.